### PR TITLE
fix(mp): use Angular Modules wording in howto page

### PR DESCRIPTION
**Pull request by gund**
Using Angular Module instead of Module as discussed.

### CODE REVIEW

File: docs/marketplace/dev/howtos/how-to-create-a-new-module-with-application.md [hunk 1]
```diff
@@ -1,17 +1,25 @@
 ---
-title: "HowTo: Create a new module with application"
+title: 'HowTo: Create a new Angular Module with application'
```
Comment (VadymSachenko) [inline docs/marketplace/dev/howtos/how-to-create-a-new-module-with-application.md:2]: @gund, why do you have "Module" uppercase?

Comment (gund) [reply docs/marketplace/dev/howtos/how-to-create-a-new-module-with-application.md:2]: As it refers to the term for consistency I've used uppercase for it.

Comment (VadymSachenko) [reply docs/marketplace/dev/howtos/how-to-create-a-new-module-with-application.md:2]: So, grammar-wise, the correct term, in this case, would be "Angular module".

**Review by VadymSachenko — DISMISSED** [review]

File: docs/marketplace/dev/howtos/how-to-create-a-new-module-with-application.md [hunk 2]
```diff
@@ -1,19 +1,27 @@
 ---
-title: "HowTo: Create a new module with application"
-last_updated: Aug 31, 2022
-description: This document shows how to create a new module with the application
+title: 'HowTo: Create a new Angular module with application'
```
Comment (andriitserkovnyi) [inline docs/marketplace/dev/howtos/how-to-create-a-new-module-with-application.md:2]: Why not double?

Comment (VadymSachenko) [reply docs/marketplace/dev/howtos/how-to-create-a-new-module-with-application.md]: @andriitserkovnyi fixed.

**Review by andriitserkovnyi — APPROVED** [review]

**Pull request merged**